source/jormungandr/tests/places_tests.py

```diff
@@ -240,3 +240,13 @@ def test_disruptions_in_places(self):
         assert len(places) == 1
         is_valid_places(places)
         assert len(response['disruptions']) == 0
+
+    def test_places_has_no_distance(self):
+        """So far, Kraken has no distance returned from its API (as opposed to Bragi)
```

Comment:
Minor: there should be a carriage return ;)

Reply:
there should be a warning from Black :smiling_imp: